src/components/buttons/HdLoaderButton.vue

```diff
@@ -158,18 +205,11 @@ export default {
     },
     setErrorState() {
       this.addToTransitionQue(this.draw.bind(this, this.$refs.cross));
-      this.addToTransitionQue(this.setIdleState);
       this.state = 'error';
     },
-    runTransitionQue: debounce(function debouncedQueMethod() {
-      if (this.transitionsQue.length > 0) {
-        const nextInQueMethod = this.transitionsQue.shift();
-        nextInQueMethod();
-      }
-    }, 100),
     fakeLoading() {
       const bumpLoading = () => {
-        const stopTreshold = this.isStatic ? 1 : 0.90;
+        const stopTreshold = this.isStatic ? 1 : 0.79;
```

Comment:
What does that `0.79` stand for?
Could it be stored in a semantic named variable?

Reply:
`0.79`  :  the stop threshold of fake loading transition ( 79% of the full circle )
yes, you are right it would be better to have it in a semantically named variable.